tests/snappy_pipeline/workflows/test_workflows_ngs_mapping.py

```diff
@@ -551,15 +551,23 @@ def test_gatk_post_bam_step_part_get_output_files(ngs_mapping_workflow):
 
 def test_gatk_post_bam_step_part_get_log_file(ngs_mapping_workflow):
     """Tests GatkPostBamStepPart.get_log_file()"""
-    expected = "work/{mapper}.{library_name}/log/snakemake.gatk_post_bam.log"
+    base_name_out = "work/{mapper}.{library_name}/log/gatk_post_bam.{library_name}"
```

Comment:
There is a method to build the expected log files. Please use: `get_expected_log_files_dict(base_out)`